mock/main.go

```diff
@@ -6,12 +6,36 @@ import (
 	"github.com/extism/go-pdk"
 )
 
+//go:export reflectMapHost
+func reflectMapHost(kPtr uint64) uint64 {
+	kMem := pdk.FindMemory(kPtr)
+	k := string(kMem.ReadBytes())
+
+	var obj map[string]interface{}
+	err := pdk.JSONFrom(kPtr, &obj)
+	if err != nil {
+		pdk.SetError(err)
+		return 0
+	}
+
```

Comment:
Now that we link the test imports (e.g. assert, etc) into XTP Tests, I wonder if it would be worthwhile to run assertions on the kv pairs, doing a type assertion inside the mock here too? 

e.g.

```go
import (
    xtptest "github.com/dylibso/xtp-test-go"
)
...

a, ok := obj["keyFromJs"].(int)
xtptest.AssertEq("reflectMapHost preserved map int value", a, 42)

...
```

Reply:
I _think_ this should work, assuming all the xtp CLI PRs are merged and the latest version is relased